ui/lib/kmip/addon/components/kmip/role-form.hbs

```diff
@@ -0,0 +1,82 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form {{on "submit" (perform this.save)}}>
+  <MessageError @model={{@model}} data-test-edit-form-error />
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <NamespaceReminder @mode="save" />
+    {{#if (eq @mode "create")}}
```

Comment:
Instead of passing `@mode` do we want to leverage `@model.isNew`, since that's been the newer pattern we've used?